srcpkgs/solvespace/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'solvespace'
+pkgname=solvespace
+version=2.3
+revision=1
+wrksrc=solvespace
+only_for_archs="i686 x86_64 x86_64-musl"
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release"
+hostmakedepends="pkg-config"
+makedepends="glew-devel gtkmm2-devel json-c-devel"
+short_desc="Parametric 2d/3d CAD"
+maintainer="Robert Lowry <bobertlo@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="http://solvespace.com/"
+distfiles="https://github.com/solvespace/solvespace/releases/download/v${version}/solvespace-source.tar.gz"
+checksum=0be0b2100c0b01eddded2a4dc60455d61cf67816189905af0e73082bfef96000
+
+pre_configure() {
+	sed -i'' -e's/CHAR_WIDTH/CHAR_WIDTH_/g' src/ui.h src/textwin.cpp src/toolbar.cpp
```

Comment:
I would rather it ship with a patch (https://github.com/solvespace/solvespace/commit/7eb6574f90310f9eb294e483b0ec9ed976a66e8e.patch), so that we know *why* the change is needed and so the patch is dropped on the next release.